Tests/test_imagesequence.py

```diff
@@ -60,10 +60,9 @@ def test_tiff() -> None:
 
 
 @skip_unless_feature("libtiff")
-def test_libtiff() -> None:
-    TiffImagePlugin.READ_LIBTIFF = True
+def test_libtiff(monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.setattr(TiffImagePlugin, "READ_LIBTIFF", True)
     _test_multipage_tiff()
```

Comment:
While not as useful, these two tests can also be combined into one with `pytest.mark.parametrize` for slightly shorter code.